### PR TITLE
feat/metadataDefaultPolygon

### PR DESCRIPTION
**Pull request by mouadhbb**

### CODE REVIEW

**Review by hichem-makhlouf — APPROVED** [review]

File: functions/src/routers/metadata-router.express.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+      if (!errors.isEmpty()) {
+        return res.status(400).json({errors: errors.array()});
+      }
+      // @ts-ignore
+      const {instanceId, tokenId} = req.params;
+      functions.logger.info("GET /metadata/polygon/:instanceId/:tokenId");
+      return res.status(200).send({
+        name: `ArianeePassport ${tokenId}`,
+        description: "TBD",
+        image: "https://cert.arianee.net/this-nft-is-set-to-private.svg",
```
Comment (mvaullerin) [inline functions/src/routers/metadata-router.express.ts:22]: should be a PNG? and https://cert.arianee.org/assets/nft-default-metadata.png

File: firebase.json [hunk 2]
```diff
@@ -6,46 +6,56 @@
...
+            "key": "Access-Control-Allow-Methods",
+            "value": "POST, GET, OPTIONS, HEAD"
+          },
+          {
+            "key": "Access-Control-Allow-Headers",
+            "value": "*"
+          }
+        ]
+      }
+    ]
```
Comment (mvaullerin) [inline firebase.json:53]: I think you need to add :
"rewrites": [ {
      "source": "/metadata/**",
      "function": "metadata"
    },

**Review by mvaullerin — CHANGES_REQUESTED** [review]

File: functions/src/routers/metadata-router.express.ts (hunk 1 above)
Comment (mouadhbb) [reply functions/src/routers/metadata-router.express.ts]: done

File: firebase.json (hunk 2 above)
Comment (mouadhbb) [reply firebase.json]: done

**Review by mvaullerin — APPROVED** [review]

**Pull request merged**